### PR TITLE
Se translation updates

### PR DESCRIPTION
**Pull request by ulfhedlund**
Summary of changes:
Added missing strings for Save and delete theme plus some minor corrections

### CODE REVIEW

File: radio/src/translations/se.h [hunk 1]
```diff
@@ -344,8 +344,8 @@
 #define TR_NAME                         "Namn"
 #define TR_MODELNAME                    "Modellnamn"
 #define TR_PHASENAME                    "Lägesnamn"
-#define TR_MIXNAME                      TR("Mixer","Mixernamn")
-#define TR_INPUTNAME                    TR("Input","Inputnamn")
+#define TR_MIXNAME                      TR("Namn","Mixernamn")
+#define TR_INPUTNAME                    TR("Namn","Inputnamn")
```
Comment (pfeerick) [inline radio/src/translations/se.h:348]: I'll let this go for now, but I'm not entirely convinced it is the right way to be doing this... i.e. might as well just use `TR_NAME` if going to lose the specificity of the translation string. I think the B&W UI needs a little rework around the inputs page as you have the input name (which gets used in the rest of the UI) and also a line name that gets shown in the inputs list... and that latter detail isn't very clear when it just lists it as "Input" and "Name"...

**Comment by ulfhedlund** [comment]
I agree that the bw UI could do with some rework. I thought the context made it clear what the name was for and that it was slightly confusing to have "Input" and "Mixer" for the line name. But I have no strong opinion and can change it back as it was before. The longer strings "Input name" etc isn't used on the color radios so isn't really needed, I believe.Den 15 feb. 2023 04:01 skrev Peter Feerick ***@***.***>:

@pfeerick commented on this pull request.







In radio/src/translations/se.h:

> @@ -344,8 +344,8 @@

 #define TR_NAME                         "Namn"

 #define TR_MODELNAME                    "Modellnamn"

 #define TR_PHASENAME                    "Lägesnamn"

-#define TR_MIXNAME                      TR("Mixer","Mixernamn")

-#define TR_INPUTNAME                    TR("Input","Inputnamn")

+#define TR_MIXNAME                      TR("Namn","Mixernamn")

+#define TR_INPUTNAME                    TR("Namn","Inputnamn")



I'll let this go for now, but I'm not entirely convinced it is the right way to be doing this... i.e. might as well just use TR_NAME if going to lose the specificity of the translation string. I think the B&W UI needs a little rework around the inputs page as you have the input name (which gets used in the rest of the UI) and also a line name that gets shown in the inputs list... and that latter detail isn't very clear when it just lists it as "Input" and "Name"...

**Comment by pfeerick** [comment]
> The longer strings "Input name" etc isn't used on the color radios so isn't really needed, I believe

No, with a two parameter TR string, second string *should* be used by colorlcd, and any screen with 212px or wider screen (so also taranis x9d B&W radios).... this is not the same as strings that are ONLY used by colorlcd. 
image

> I agree that the bw UI could do with some rework. I thought the context made it clear what the name was for and that it was slightly confusing to have "Input" and "Mixer" for the line name. 

No need to change it. For your translation, yes it works, in mine (EN) it wouldn't, as then I'd have two lines saying "Name" 🤪 In yours the Expo one is different to mine... therein my gripe here is more about translations diverging... i.e. as a programmer, I would **expect**, for example,  `TR_MIXNAME` to be the same for all languages, so that when I use it, it means the same thing for all languages. In this case, `TR_MIXNAME`, `TR_INPUTNAME` and `TR_EXPONAME` are all over the place across the translations. Plus, why have them if could simply be using using `TR_NAME` instead. :wink:

**Comment by ulfhedlund** [comment]
For color radios TR_INPUTNAME string is used for Inputs, whereas for Mixers TR_NAME is used, not TR_MIXERNAME. 
For B&W radios TR_INPUTNAME and TR_MIXERNAME is used. 
Perhaps that should be changed - I'm not sure.

In any case, I'll leave these translations it for now but may change them later on.

**Pull request merged**